docs/tutorials/tutorial-sketches-theta.md

```diff
@@ -0,0 +1,322 @@
+
+### Filtered metrics
+
+Druid has the capability to use [filtered metrics](../querying/sql-aggregations.md). This means you can include a WHERE clause in the SELECT part of the query.
+> In the case of Theta sketches, the filter clause has to be inserted between the aggregator and the estimator.
+
+As an example, query the total unique users that watched _Bridgerton:_
+
+```sql
+SELECT THETA_SKETCH_ESTIMATE(
+         DS_THETA(theta_uid) FILTER(WHERE "show" = 'Bridgerton')
+       ) AS users
+FROM ts_tutorial
+```
+
+![Count distinct with Theta sketches and filters](../assets/tutorial-theta-08.png)
+
+### Set operations
+
+You can use this capability of filtering in the aggregator, together with _set operations_, to finally answer the questions from the introduction.
+
+How many users watched both episodes of _Bridgerton?_ Use `THETA_SKETCH_INTERSECT` to compute the unique count of the intersection of two (or more) segments:
+
+```sql
+SELECT THETA_SKETCH_ESTIMATE(
+         THETA_SKETCH_INTERSECT(
+           DS_THETA(theta_uid) FILTER(WHERE "show" = 'Bridgerton' AND "episode" = 'S1E1'),
+           DS_THETA(theta_uid) FILTER(WHERE "show" = 'Bridgerton' AND "episode" = 'S1E2')
+         )
+       ) AS users
+FROM ts_tutorial
+```
+
+![Count distinct with Theta sketches, filters, and set operations](../assets/tutorial-theta-09.png)
+
+Again, the set function is spliced in between the aggregator and the estimator.
+
+Likewise, use `THETA_SKETCH_UNION` to find the number of visitors that watched _any_ of the episodes:
+
+```sql
+SELECT THETA_SKETCH_ESTIMATE(
+         THETA_SKETCH_UNION(
+           DS_THETA(theta_uid) FILTER(WHERE "show" = 'Bridgerton' AND "episode" = 'S1E1'),
+           DS_THETA(theta_uid) FILTER(WHERE "show" = 'Bridgerton' AND "episode" = 'S1E2')
+         )
+       ) AS users
+FROM ts_tutorial
+```
+
+![Count distinct with Theta sketches, filters, and set operations](../assets/tutorial-theta-10.png)
+
+And finally, there is `THETA_SKETCH_NOT` which computes the set difference of two or more segments.
+The result describes how many visitors watched episode 1 of Bridgerton but not episode 2.
+
+
+```sql
+SELECT THETA_SKETCH_ESTIMATE(
+         THETA_SKETCH_NOT(
+           DS_THETA(theta_uid) FILTER(WHERE "show" = 'Bridgerton' AND "episode" = 'S1E1'),
+           DS_THETA(theta_uid) FILTER(WHERE "show" = 'Bridgerton' AND "episode" = 'S1E2')
+         )
+       ) AS users
+FROM ts_tutorial
+```
+
+![Count distinct with Theta sketches, filters, and set operations](../assets/tutorial-theta-11.png)
+
+## Conclusions
+
+- Counting distinct things for large data sets can be done with Theta sketches in Apache Druid.
+- This allows us to use rollup and discard the individual values, just retaining statistical approximations in the sketches.
+- With Theta sketch set operations, affinity analysis is easier, for example, to answer questions such as which segments correlate or overlap by how much.
+
+## Further reading
```

Comment:
any reason we're not using "Learn more" ? 😬

Reply:
All the Druid tutorials that have this section call it "Further reading" so this keeps it in alignment